core/server/master/src/main/java/alluxio/master/meta/checkconf/InconsistentProperty.java

```diff
@@ -24,10 +27,10 @@
  */
 public class InconsistentProperty {
   /** The name of the property that has errors/warnings. */
-  private String mName;
+  private String mName = "";
   /**
    * Record the values and corresponding hostnames. */
-  private Map<String, List<String>> mValues;
+  private Map<String, List<String>> mValues = new HashMap<>();
```

Comment:
NIT: `final`

Reply:
I cannot use final here. The actual name and values will be set via setName and setValues methods. The initialize name and values here are to avoid NPE via Object.equals function.

Reply:
ack